### PR TITLE
glog v0.6 files

### PR DESCRIPTION
**Pull request by JaiXJM-BB**
Tests:
[glog_7.1_x86_64_test.txt](https://github.com/user-attachments/files/18116616/glog_7.1_x86_64_test.txt)
[glog_8.0_aarch64le_test.txt](https://github.com/user-attachments/files/18116617/glog_8.0_aarch64le_test.txt)
Unsure if core dump is unintended or a necessary consequence of testing logs of failure states.

### CODE REVIEW

**Comment by eleir9268** [comment]
Was it a subprocess that died to produce the core dump?

**Comment by JaiXJM-BB** [comment]
> Was it a subprocess that died to produce the core dump?

Yes, I call the tests as subprocesses. That specific test is for logging crash conditions like nullpointer access etc so I figured it might have been intended? The tests fully complete first. I'll compile it for linux and check if the same thing happens.

**Comment by eleir9268** [comment]
If the test harness itself is not dying (i.e. it's spawning a child to die and capture the results) then it's fine. There are a lot of tests that cover that kind of thing and we don't have time to check all of them.

**Comment by JaiXJM-BB** [comment]
OK, in that case I think we are good.

File: ports/gflags/common.mk [hunk 1]
```diff
@@ -32,7 +34,10 @@ CFLAGS += $(FLAGS)
 include $(MKFILES_ROOT)/qtargets.mk
 
 CMAKE_ARGS = -DCMAKE_TOOLCHAIN_FILE=$(PROJECT_ROOT)/qnx.nto.toolchain.cmake \
-             -DCMAKE_INSTALL_PREFIX=$(gflags_INSTALL_ROOT)/$(CPUVARDIR)/usr \
+             -DCMAKE_INSTALL_PREFIX=$(INSTALL_ROOT)/$(CPUVARDIR)/$(PREFIX) \
+             -DCMAKE_INSTALL_LIBDIR=$(INSTALL_ROOT)/$(CPUVARDIR)/$(PREFIX)/lib \
+             -DCMAKE_INSTALL_BINDIR=$(INSTALL_ROOT)/$(CPUVARDIR)/$(PREFIX)/bin \
+			 -DCMAKE_INSTALL_INCLUDEDIR=$(INSTALL_ROOT)/$(PREFIX)/include \
```
Comment (eleir9268) [inline ports/gflags/common.mk:40]: Whitespace issue.

File: ports/glog/common.mk [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+             -DCMAKE_BUILD_TYPE=$(CMAKE_BUILD_TYPE) \
+             -DCMAKE_SYSTEM_PROCESSOR=$(CPUVARDIR) \
+             -DEXTRA_CMAKE_C_FLAGS="$(CFLAGS)" \
+             -DEXTRA_CMAKE_CXX_FLAGS="$(CFLAGS)" \
+             -DEXTRA_CMAKE_ASM_FLAGS="$(FLAGS)" \
+             -DEXTRA_CMAKE_LINKER_FLAGS="$(LDFLAGS)" \
+			 -DCMAKE_DEBUG_POSTFIX="" \
+			 -Dgflags_DIR=$(INSTALL_ROOT)/$(CPUVARDIR)/$(PREFIX)/lib/cmake/gflags \
+			 -DGTest_DIR=$(INSTALL_ROOT)/$(CPUVARDIR)/$(PREFIX)/lib/cmake/GTest \
+			 -DBUILD_TESTING=ON
```
Comment (eleir9268) [inline ports/glog/common.mk:49]: Whitespace issue.

File: ports/glog/common.mk [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+MAKE_ARGS ?= -j $(firstword $(JLEVEL) 1)
+
+ifndef NO_TARGET_OVERRIDE
+glog_all:
+	@mkdir -p build
+	@cd build && cmake $(CMAKE_ARGS) $(DIST_BASE)/
+	@cd build && make VERBOSE=1 all $(MAKE_ARGS)
+
+install check:
```
Comment (eleir9268) [inline ports/glog/common.mk:59]: You *probably* need the glog_all rule dependency here.

```
install check: glog_all
```

**Review by eleir9268 — CHANGES_REQUESTED** [review]

File: ports/glog/common.mk (hunk 3 above)
Comment (JaiXJM-BB) [reply ports/glog/common.mk]: Implemented with b5f057fcda0ae990b86b13855dff2fae6aa5615e :)

File: ports/glog/common.mk (hunk 2 above)
Comment (JaiXJM-BB) [reply ports/glog/common.mk]: Fixed with b5f057fcda0ae990b86b13855dff2fae6aa5615e - set to all spaces.

File: ports/gflags/common.mk (hunk 1 above)
Comment (JaiXJM-BB) [reply ports/gflags/common.mk]: Fixed with b5f057fcda0ae990b86b13855dff2fae6aa5615e - set to all spaces.

**Review by eleir9268 — APPROVED** [review]

**Comment by JaiXJM-BB** [comment]
^resolving merge conflict

**Pull request merged**